### PR TITLE
Make generate-schema-imports.sh portable & stable

### PR DESCRIPTION
**Pull request by fomentia**

### CODE REVIEW

File: tools/generate-schema-imports.sh [hunk 1]
```diff
@@ -2,14 +2,21 @@
...
+    sed -i"" -e "\|$MARKER|,$ {" -e "\|$MARKER|b" -e "${1}d" -e "}" src/IntegrationSchema.ts
+}
 
-for SCHEMA_FILENAME in $(ls src/schemas | sort); do
-    SCHEMA=$(echo $SCHEMA_FILENAME | cut -d . -f 1)
-    sed -i "" "\|$MARKER|a\\
-import $SCHEMA from \"./schemas/$SCHEMA_FILENAME\";\\
+sed_delete_after_marker
+
+for SCHEMA in $(ls src/schemas | sed "s/\.json$//" | sort -f); do
```
Comment (fomentia) [inline tools/generate-schema-imports.sh:11]: `sort -f`: case-insensitive.

**Comment by fomentia** [comment]
@aiwilliams Pls run on Mac to verify stability. :heart:

**Comment by fomentia** [comment]
This is not working due to the `-i` option for `sed` being formatted differently between MacOS's and GNU's versions (`-i "ext"` on MacOS, `-i"ext"` or `--in-place="ext"` on GNU). :disappointed:

**Comment by aiwilliams** [comment]
Going to close this since it doesn't work on both platform.

**Pull request closed without merging**